src/components/RenderHTML/index.native.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
```

Comment:
Why disable `react/prop-types`? Instead, import the prop types from the prop types file.